NBitcoin.Altcoins/ForkIdTransaction.cs

```diff
@@ -47,7 +47,7 @@ public override uint256 GetSignatureHash(Script scriptCode, int nIn, SigHash nHa
 			if(UsesForkId(nHashType))
 				nForkHashType |= ForkId << 8;
 
-			if((SupportSegwit && sigversion == HashVersion.Witness) || UsesForkId(nHashType))
```

Comment:
Keep ` || UsesForkId(nHashType)` please, I don't want to test again bcash and bgold.